docs/static/transforming-data.asciidoc

```diff
@@ -359,8 +359,22 @@ TIP: If you need help building grok patterns, try out the
 [[lookup-enrichment]]
 === Enriching Data with Lookups
 
-The plugins described in this section are useful for enriching data with
-additional info, such as GeoIP and user agent info.
+These plugins can help you enriching data with
+additional info, such as GeoIP and user agent info:
+
+* dns filter
```

Comment:
Agreed, though I thought these were already linked to the respective plugin docs pages.

Reply:
@acchen97  You're right! The plugin names were/are set up as links  as of this commit.   
I added the unordered list at the top so that people could quickly get a sense of all lookup plugins available instead of scrolling up/down the page to see them all.  @jsvd I agree that the list isn't as helpful as it could be if it had links. 

So here's what I've come up with.  With the next commit you'll see that I made the mini-TOC items link to the plugin descriptions on this page.  (That's how a user would likely expect a mini-TOC at the top of a page to work.) Within each plugin description, there's a link to take you to the plugin doc.  It's not as weird as it sounds. :-)  I can demo and/or we can discuss.